code/src/assets/icons/CartOutlined.tsx

```diff
@@ -0,0 +1,51 @@
+import { ComponentProps } from "react";
+
+const CartOutlined = (props: ComponentProps<"svg">) => (
+  <svg
+    version="1.1"
+    id="Шар_1"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlnsXlink="http://www.w3.org/1999/xlink"
+    height="28px"
+    width="28px"
+    viewBox="0 0 12 12"
+    xmlSpace="preserve"
+    {...props}
+  >
+    <g>
+      <path
```

Comment:
Using `svg` as component is generally considered bad approach because it's code goes to javascript bundle. We should try to store it inside assets or use package for it, for example for `vite` it is `vite-plugin-svgr`, I think there should be package for `webpack` also. As far as I can see, these components do not depend from props, so we can move it to assets.

Recently I came up with interesting video and I strongly recommend to watch it. Here is the link: https://www.youtube.com/watch?v=5r9sN5Yz79A

I hope it will be helpful for you

Reply:
Thank you for suggestion and for video, I found out new information for myself. I changed it in favour of the approach shown in video.

Reply:
You are welcome. I am happy to hear that it was useful for you